doc/contributing/hw/comportability/README.md

```diff
@@ -462,62 +462,137 @@ Here is an example specification:
 
 ## Interrupt Handling
 
-Interrupts are critical and common enough to attempt to standardize across the project.
-Where possible (exceptions for inherited IP that is too tricky to convert) all interrupts shall have common naming, hardware interface, and software interface.
-These are described in this section.
+Interrupts are critical and common enough to standardize across the project.
+Where possible (exceptions for inherited IP that is too tricky to convert) all interrupts shall have common naming, hardware interfaces, and software interfaces.
 
-### Event Type Interrupt
+### Conventions
 
-Event type interrupts are latched indications of defined peripheral events that have occurred and not yet been addressed by the local processor.
-All interrupts are sent to the processor as active-high level (as opposed to edge) interrupts.
-Events themselves can be edge or level, active high or low, as defined by the associated peripheral.
-For instance, the GPIO module might detect the rising or falling edge of one its input bits as an interrupt event.
+Currently, all wired CIP interrupts in OpenTitan use active-high level-triggering.
+(An alternate standard is the use of edge-triggered interupts, which are not described here, but may be supported in a future release.)
 
-The latching of the event is done by the auto-generated register file as described below.
-The clearing of the event is done by a processor write when the handling of the event is completed.
-The waveform below shows the timing of the event occurrence, its latched value, and the clearing by the processor.
-More details follow.
+![OpenTitan System Interrupt Architecture](ot_interrupt_arch.svg)
+
+Interrupts sent to the processor are aggregated by a platfrom level interrupt controller (PLIC).
+Within that logic there may be another level of control for enabling, prioritizing, and enumeration.
+Specification of this control is defined in the rv_plic documentation of the corresponding toplevel design.
```

Comment:
typo on L-475: platform
Should we link the rvplic spec directly (https://opentitan.org/book/hw/top_earlgrey/ip_autogen/rv_plic/) and if we do that drop L-477?

Reply:
I guess the problem is that this document is supposed to be more general (and not only apply to top_earlgrey and its choice of rv_plic). I *think* I probably like the way that Harry has done it so far.

Reply:
Yes that's right, rv_plic is instantiated using ipgen (https://github.com/lowRISC/opentitan/pull/8431) and ideally here we could link to a generic, unparameterized version of the block. Unfortunately, we don't yet serve the docs for the block template (opentitan/hw/ip_templates/rv_plic/).
I think for now I will leave it without a link, but in the future we could add one.